release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,32 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 30 August 2024 - Enhancement - Catalog - Type filter documentation link [ID_40601]
+
+When you search for an item in the Catalog, a documentation link will now be shown next to the type filter. This link will navigate to the Catalog types information on DataMiner Docs.
+
+#### 30 August 2024 - Enhancement - Catalog - Owners or source no longer shown in details page side panel when not defined [ID_40584]
+
+On the details page of a Catalog item, the side panel will no longer show the owners and source section if the owners or source are not defined.
+
+#### 30 August 2024 - Fix - Catalog - Search overlay on Catalog details not closing [ID_40533]
+
+When you searched for a Catalog item while on a Catalog detail page, it could occur that the search overlay did not close when the search results were loaded.
+
+#### 30 August 2024 - Enhancement - Admin - Improved restrictions for user role changes [ID_40526]
+
+The user role is now disabled for user that are not allowed to change roles.
```

Comment:
I'm not entirely sure how to interpret this. Is this what you mean?
```suggestion
In the Admin app, users who do not have the rights to change roles will now no longer be able to select other user roles.
```

Reply:
Yes, that is correct.